client/app/telecom/telephony/alias/special/fees/translations/Messages_fr_FR.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8"?>
+<translations>
+   <translation id="telephony_alias_special_fees_title_back_link">Retour à la gestion des reversements</translation>
+   <translation id="telephony_alias_special_fees_title">Coûts SVA</translation>
+   <translation id="telephony_alias_special_fees_date_filter">Date</translation>
+   <translation id="telephony_alias_special_fees_calling_filter">Appelant</translation>
+   <translation id="telephony_alias_special_fees_duration_filter">Durée (sec.)</translation>
+   <translation id="telephony_alias_special_fees_operator_filter">Opérateur</translation>
+   <translation id="telephony_alias_special_fees_price_filter">Montant (€)</translation>
+   <translation id="telephony_alias_special_fees_total">Total</translation>
+   <translation id="telephony_alias_special_fees_calls">{{calls}} appel{{calls > 1 ? 's': ''}}</translation>
```

Comment:
`ngPluralize` probably help you in that case.
see: https://docs.angularjs.org/api/ng/directive/ngPluralize